test/integration/font-optimization/test/index.test.js

```diff
@@ -96,12 +96,10 @@ describe('Font Optimization', () => {
           const link = $(
             `link[rel="stylesheet"][data-href="${staticHeadFont}"]`
           )
-          const nonce = link.attr('nonce')
           const style = $(`style[data-href="${staticHeadFont}"]`)
           const styleNonce = style.attr('nonce')
 
-          expect(link).toBeDefined()
-          expect(nonce).toBe('VmVyY2Vs')
+          expect(link).toBeUndefined()
```

Comment:
The <link> tag will be removed from the resultant HTML when font-optimisation is enabled. Hence, assert that it is undefined.